assets/helpers/payPalContributionsCheckout/payPalContributionsCheckout.js

```diff
@@ -4,46 +4,82 @@
 
 import { participationsToAcquisitionABTest, getOphanIds } from 'helpers/tracking/acquisitions';
 import * as cookie from 'helpers/cookie';
-import { addQueryParamToURL } from 'helpers/url';
+import { addQueryParamToURL, getAbsoluteURL } from 'helpers/url';
+import { routes } from 'helpers/routes';
 
 import type { IsoCountry } from 'helpers/internationalisation/country';
 import type { OphanIds, AcquisitionABTest, ReferrerAcquisitionData } from 'helpers/tracking/acquisitions';
 import type { Participations } from 'helpers/abTests/abtest';
 import { countryGroups } from 'helpers/internationalisation/countryGroup';
 import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
+import type { IsoCurrency } from 'helpers/internationalisation/currency';
 
 // ----- Types ----- //
 
-type PayPalPostData = {|
-  countryGroup: string,
-  amount: number,
-  cmp: ?string,
-  intCmp: ?string,
-  refererPageviewId: ?string,
-  refererUrl: ?string,
-  ophanPageviewId: string,
-  ophanBrowserId: ?string,
-  ophanVisitId: ?string,
-  supportRedirect: boolean,
+type AcquisitionData = {|
+  pageviewId: string,
+  visitId: ?string,
+  browserId: ?string,
+  platform: ?string,
+  referrerPageviewId: ?string,
+  referrerUrl: ?string,
+  campaignCodes: ?string[],
   componentId: ?string,
   componentType: ?string,
   source: ?string,
-  refererAbTest: ?AcquisitionABTest,
-  nativeAbTests: ?AcquisitionABTest[],
+  abTests: ?AcquisitionABTest[],
+|}
+
+type PayPalPaymentAPIPostData = {|
+  currency: IsoCurrency,
+  amount: number,
+  returnURL: string,
+  cancelURL: string,
 |}
 
 // ----- Functions ----- //
 
 function payalContributionEndpoint(testUser) {
```

Comment:
Can we fix this typo while we're here? (payal?)

Reply:
Fixed in ecb4226ac439c8e8bfe5ec3cc7cfca403f22b047 Good catch @JustinPinner!!!